common/testing/testlogger/testlogger.go

```diff
@@ -204,18 +204,6 @@ func LogLevel(level zapcore.Level) LoggerOption {
 	}
 }
 
-func WithoutCaller() LoggerOption {
-	return func(t *TestLogger) {
-		t.state.logCaller = false
-	}
-}
-
-func LogCaller() LoggerOption {
-	return func(t *TestLogger) {
-		t.state.logCaller = false
-	}
-}
-
```

Comment:
This doesn't work together with  `log.BuildZapLogger` which defaults to disabling this. Removing it for now.